frontend/src/components/App.js

```diff
@@ -201,15 +205,15 @@ const App = ({userID, usersByID, logOut, get}) => {
             width={224}
             style={{overflow: 'auto'}}
           >
-            <div style={{display: 'flex', alignContent: 'baseline', justifyContent: 'left', textAlign: 'center'}}>
+            <div style={{alignContent: 'baseline', justifyContent: 'left', textAlign: 'center', background: env !== 'PROD' ? notProdBanner : undefined}}>
               <Title style={titleStyle} className="App__title">
                 <div>
-                  <b>F</b><span>reeze</span><b>M</b><span>an</span>
+                  <span>FreezeMan</span>{env !== 'PROD' && <span style={{color: 'red'}}>&nbsp;{env}</span>}
                 </div>
               </Title>
               { // Display a spinner while the initial data is being fetched at startup 
                 !isInitialized &&
-                  <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}}>
+                  <div style={{position: 'absolute', left: '0', right: '0', top: env !== 'PROD' ? '3.25rem' : '3.5rem'}}>
```

Comment:
I think it would be better to find a way to allow CSS to position the spinner, rather than using absolute position and a hard-coded offset from the top. An ant grid component with 3 rows (top one empty)  and 1 column might work for this.

Reply:
I've attempted that. It's painful, and the spinner ends up behind the jump bar :(

Reply:
This was my attempt. (thanks to ChatGPT)

```jsx
      <Row justify="center" align="middle" style={{ height: '100%' }}>
        <Col span={24}>
          <Title style={titleStyle} className="App__title">
            <span>FreezeMan</span>
            {env !== 'PROD' && <span style={{ color: 'red' }}>&nbsp;{env}</span>}
          </Title>
        </Col>
      </Row>
      <Row justify="center" align="middle" style={{ height: '100%' }}>
        <Col span={24}>
          {/* Display a spinner while the initial data is being fetched at startup */}
          <Spin
            size="small"
            indicator={loadingIcon}
            style={{ visibility: !isInitialized ? undefined : 'hidden' }}
          />
        </Col>
      </Row>
```